ci-operator/config/openshift/openstack-cinder-csi-driver-operator/openshift-openstack-cinder-csi-driver-operator-master.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+binary_build_commands: go build cmd/openstack-cinder-csi-driver-operator/main.go
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: openstack-cinder-csi-driver-openstack
+promotion:
+  name: "4.6"
```

Comment:
Should it be 4.7?

Reply:
This part is generated by `make new-repo`, I believe it will be changed when 4.7 master is open